src/main/java/Program.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Objects;
+
+public class Program {
+    private boolean canTerminateHal = false;    //when true, the program exits
+    private static Task[] listTasks = new Task[999];
+    private static int numItems;
+
+    private static int DEADLINE_INDEX = 9;
+    private static int EVENT_INDEX = 5;
+    private static int TODO_INDEX = 4;
+    private static int TASK_STRING_OFFSET = 3;
+
+    public static final String LINE_BREAK_SINGLE = "____________________________________________________________";
+    public static final String TASK_ADDED_DONE_TEXT = "Got it! I've added this task: ";
+    public static final String DONE_TASK_ERROR_MESSAGE = "No such task exist! Are you sure you keyed in the correct number?";
+    public static final String DONE_TASK_SUCCESS_MESSAGE = "Nice! I've marked this task as done:";
+    public static final String PRINT_ERROR_MESSAGE = "Your input does not follow my format!\n" +
+            "Read properly and type it again!";
+    public static final String ENTER_COMMAND_TEXT = "Enter command: ";
+
+    public Program() {
+        this.numItems = 0;
+    }
+
+    public static int getNumItems() {
+        return numItems;
+    }
+
+    //function to parse input text by the user and run corresponding function
+    public void executeTask(String string) {
```

Comment:
Perhaps could use more intuitive name to define what exactly string is?

Reply:
Okay that makes sense, thanks!